src/elexmodel/models/BaseElectionModel.py

```diff
@@ -6,6 +6,8 @@
 import cvxpy
 import numpy as np
 from elexsolver.QuantileRegressionSolver import QuantileRegressionSolver
+from sklearn.metrics import mean_absolute_percentage_error
+from sklearn.model_selection import KFold
```

Comment:
I'm curious if there is a way to do this without using scikit-learn, which is a pretty large library and would add significantly to the overhead of this repo. For `mean_absolute_percentage_error` we can use `elexmodels.math_utils.compute_error(true, pred, type_='mape')`. `KFold` might be a bit harder to implement, but still worth it I think.